File: CompSci 1/Unit 6, Scanner Class/Rectangle/Rectangle.java
import java.util.Scanner;

public class Rectangle {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter the Length --> ");
        int length = sc.nextInt();

        System.out.print("Enter the Width --> ");
        int width = sc.nextInt();

        System.out.println("Perimeter = " + (2 * length + 2 * width));
        System.out.println("Area = " + length * width);
    }
}

File: CompSci 1/Unit 6, Scanner Class/Rectangle/Rectangle.kt
import java.util.Scanner

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`)

    print("Enter the Length --> ")
    val length = sc.nextInt()

    print("Enter the Width --> ")
    val width = sc.nextInt()

    println("Perimeter = " + (2 * length + 2 * width))
    println("Area = " + length * width)
}